e-vent.ino

```diff
@@ -232,6 +232,7 @@ void loop() {
       if (enteringState) {
         enteringState = false;
         goToPositionByDur(roboclaw, BAG_CLEAR_POS, motorPosition, tEx - (now() - tCycleTimer));
+        displ.hideIcon(0,19);
```

Comment:
Low level display information such as icon coordinates shouldn't go here

Reply:
Revised by moving the location details to the display